src/player.cpp

```diff
@@ -4572,13 +4572,17 @@ int player::throw_range(int pos)
  // Increases as weight decreases until 150 g, then decreases again
  int ret = (str_cur * 8) / (tmp.weight() >= 150 ? tmp.weight() / 113 : 10 - int(tmp.weight() / 15));
  ret -= int(tmp.volume() / 4);
+ bool is_using_atlatl = weapon.type_name() == "atlatl";
```

Comment:
You probably want to use `typeId()` here because `type_name` returns the translated item name, which will be different when the player uses a translation.